src/function.rs

```diff
@@ -4,7 +4,9 @@ use {
     ToKebabCase, ToLowerCamelCase, ToShoutyKebabCase, ToShoutySnakeCase, ToSnakeCase, ToTitleCase,
     ToUpperCamelCase,
   },
+  rand::prelude::*,
```

Comment:
I don't know how people feel about * imports, but if we could avoid it that would be great!

Reply:
That's normally true, but `rand::prelude::*` is designed to be imported with a wildcard like this.

Reply:
Yeah -- preludes are always optional though, I think it's a stylistic/feelings based choice. While the *quick start* section of `rand` and the `rand` book use the prelude import, that's not exclusively how they use it (ex in the ["types of generators" section](https://rust-random.github.io/book/guide-gen.html))